src/Microsoft.AspNet.Razor.Runtime/TagHelpers/TagHelperExecutionContext.cs

```diff
@@ -22,12 +22,13 @@ public class TagHelperExecutionContext
         /// <summary>
         /// Internal for testing purposes only.
         /// </summary>
-        internal TagHelperExecutionContext(string tagName)
+        internal TagHelperExecutionContext(string tagName, bool selfClosing)
             : this(tagName,
```

Comment:
it's `internal` so changing the signature wasn't required.  but you've done the work and it's not harmful.  I vote for leaving this change in.